shared/chat/inbox/row/small-team/swipe-conv-actions/index.native.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters/mobile.native'
+import * as Styles from '../../../../../styles'
+import {Props} from '.'
+
+let curSwipeRef: React.MutableRefObject<Kb.Swipeable> | null = null
```

Comment:
i think we likely prefer `RefObject` as it captures current can be null, `MutableRefObject` doesn't

Reply:
I just used that type since it is what `useRef` returns, I can try changing it.

Reply:
it actually has multiple types. you can merge types in TS: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L939